timed/employment/serializers.py

```diff
@@ -29,7 +29,7 @@ class Meta:
         """Meta information for the user serializer."""
 
         model = get_user_model()
-        fields = [
+        fields = (
```

Comment:
most likely we will want introduce this kind of change in a later type hinting focused pr
```suggestion
        fields: tuple(str, ...) = (
```
even more likely there is room for a proper type hierarchy and i need to study the typeddjango efforts to know more